layers/meta-resin-beaglebone/recipes-connectivity/bb-wl18xx-bluetooth/bb-wl18xx-bluetooth.bb

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION = "BB WL18xx Bluetooth"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
```

Comment:
It might be worth adding a comment to indicate where these were downloaded from.

Reply:
It is specified in the git log. Shall we have this info here also?

Reply:
Sorry, missed that, looks fine to me.

Reply:
Follow-up amend comig, I missed to package some files